src/Tools/ExternalAccess/EditorConfigGenerator/Microsoft.CodeAnalysis.ExternalAccess.EditorConfigGenerator.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <RootNamespace>Microsoft.CodeAnalysis.ExternalAccess.EditorConfigGenerator</RootNamespace>
+    <AssemblyName>Microsoft.CodeAnalysis.ExternalAccess.EditorConfigGenerator</AssemblyName>
```

Comment:
><AssemblyName>Microsoft.CodeAnalysis.ExternalAccess.EditorConfigGenerator</AssemblyName> [](http://example.com/codeflow?start=4&length=88)

Unnecessary since the default is the name of the project.